_config_base.yml

```diff
@@ -98,5 +95,14 @@ release_info:
     name: v22.1.5
     start_time: 2022-07-27 13:34:27.897839 +0000 UTC
     version: v22.1.5
+  v22.2:
+    build_time: 2022-08-16 10:15:48.318507 (go1.17)
+    docker_image: cockroachdb/cockroach-unstable
+    name: v22.2-alpha.1
+    start_time: 2022-08-16 10:15:48.318607 +0000 UTC
+    version: v22.2-alpha.1
```

Comment:
What's the impact of including v22.2 generally, and specifically `v22.2-alpha.1` prior to its release?

Also, not sure if related, but I am seeing an empty v22.2 section here
https://deploy-preview-14837--cockroachdb-docs.netlify.app/docs/releases/index.html

Shouldn't we disable that until the alpha release? Should we be moving some steps to the first alpha release notes PR (or a separate PR after this one but before that one)?

Reply:
That's a good note. I will update the releases index to suppress any versions that do not contain a release.

If we do not include this in the config, then the main problem we'll face is we won't see v22.2 in the version switcher. Other links may also not resolve correctly and there may be blank bits of text that use `page.version.version` anywhere in the page. I think it's necessary to keep it.

I can instead use `v22.2.0 (unreleased)` or something similar to indicate that it's currently not released. What do you think?

Reply:
> I can instead use v22.2.0 (unreleased) or something similar to indicate that it's currently not released. What do you think?

Do you mean that you'd change the value of `name` and `version` to something that's easier for us to understand, but it won't actually be shown on site? If so, that's fine by me. Maybe just `v22.2-alpha.1 (unreleased)` so the next RN PR can just drop 'unreleased'?

Reply:
I don't fully understand why we split name and version and keep them as the same thing, but whatever we set as `version` is the text that is shown directly in the version switcher drop down.

I'll take a separate follow-up action to understand the version plug-in and why it's built the way it's built.

What should we display publicly for end users?

Reply:
To recap, before alpha 1 is released, we need reasonable text for the dropdown re. what version you're seeing. Maybe

`v22.2 (dev / pre-alpha)` ?

Reply:
Edit: After discussion re. limitations, `v22.2 (dev)` is fine. Thanks!